hack/toolchain.sh

```diff
@@ -23,6 +23,8 @@ tools() {
     go install github.com/onsi/ginkgo/v2/ginkgo@latest
     go install github.com/rhysd/actionlint/cmd/actionlint@latest
     go install github.com/mattn/goveralls@latest
+    go install github.com/google/go-containerregistry/cmd/crane@latest
```

Comment:
Seems like we won't need these without the artifacthub pushing or the cosigning

Reply:
Crane is needed for cosigning, oras is needed for the artifact hub metadata.

Reply:
I think if we break out those changes into separate PRs like we discussed, then we don't need the dependencies here in the context of this PR